src/app/meta-campaign/meta-campaign.component.ts

```diff
@@ -97,7 +96,7 @@ export class MetaCampaignComponent implements AfterViewChecked, OnDestroy, OnIni
     private route: ActivatedRoute,
     public searchService: SearchService,
     private state: TransferState,
-    @Inject(TBG_DONATE_STORAGE) private storage: StorageService,
+    @Inject(SESSION_STORAGE) private storage: StorageService,
```

Comment:
I guess the thinking here is just that maintaining previous cards isn't useful after a while, rather than a PII/privacy consent issue?

I can't immediately see a problem with that, but it might be worth renaming the var for consistency with the other services, to avoid confusion should we want to expand what we store here.

Reply:
Renamed in 263848d9